.travis.yml

```diff
@@ -5,12 +5,13 @@ os: linux
 
 language: generic
 
-branches:
-  only:
-    - master
+#branches:
```

Comment:
Nit: remove or leave in place. Commented-out 'code' has no place in the master branch.

Reply:
copy that!